packages/playwright-core/src/remote/playwrightConnection.ts

```diff
@@ -58,6 +58,8 @@ export class PlaywrightConnection {
     this._preLaunched = preLaunched;
     this._options = options;
     options.launchOptions = filterLaunchOptions(options.launchOptions);
+    if (isUnderTest() && process.env.PWTEST_LAUNCH_OPTIONS_OVERRIDE)
```

Comment:
Maybe we can just turn off the filter when an environment variable is set ?

@dgozman thoughts?

Reply:
Yeah, that seems to be less plumbing and special-casing.